example/dev-proxy/templates/redirect.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>ITW - Simulate Redirect</title>
+    <script>
+      function redirectToPage() {
+        const redirectUrl = "http://192.168.1.54:8000"; // needs to be changed to your local IP
+        window.location = `${redirectUrl}/redirect`;
```

Comment:
You can use flask mapping directly
```suggestion
         window.location = "{{ url_for('auth_redirect', _external = True) }}";
```

Reply:
Nice suggestion, thanks.